components/server/src/workspace/workspace-starter.ts

```diff
@@ -222,6 +222,7 @@ export class WorkspaceStarter {
             // We may have never actually started the workspace which means that ws-manager-bridge never set a workspace status.
             // We have to set that status ourselves.
             instance.status.phase = 'stopped';
+            instance.stoppingTime = new Date().toISOString();
```

Comment:
Maybe a nit, but: guaranteed `stoppingTime == stoppedTime` is a nice thing to have for downstream usage: analytics, bug hunting, ad-hoc DB fixes, etc.